New AI get started guides (#3489)
liveblocks/liveblocks: docs/pages/get-started/nextjs-ai-presence.mdx

      You can then wrap any part of your your UI with `AiPresenceEditFrame`, give each one a unique `editingId`,
      and if the AI is editing that ID, a highlighted box will appear around it.

      ```tsx file="app/Form.tsx"
      "use client";

      import { AiPresenceEditFrame } from "./AiPresenceEditFrame";

      export function Form() {
        return (
          <form>
            // +++
            <AiPresenceEditFrame editingId="title-1">
              <input />
            </AiPresenceEditFrame>
            // +++
            <AiPresenceEditFrame editingId="description-1">
              <textarea />
            </AiPresenceEditFrame>
          </form>
        );
      }
      ```
    </StepContent>

  </Step>

  <Step lastStep>
    <StepTitle>Hide the AI when it’s done</StepTitle>
    <StepContent>

      You can optionally choose to hide the AI's presence when it's completed editing. To do
      this, set the minimum `ttl` value to 2 seconds.

      ```ts
      await liveblocks.setPresence(roomId, {
        userId: AI_USER_INFO.id,
        userInfo: AI_USER_INFO.info,
        data: { editingId: null },
        // +++
        ttl: 2,
      // +++
      });
      ```

>       You can hvw AI presence set up! In some apps, it may be useful to use multiple different IDs for AI, as this way

Reviewer: ```suggestion
      You have AI presence set up! In some apps, it may be useful to use multiple different IDs for AI, as this way
```
